src/test/java/net/openhft/chronicle/bytes/ContentEqualsJLBHTest.java

```diff
@@ -0,0 +1,73 @@
+package net.openhft.chronicle.bytes;
+
+import net.openhft.chronicle.bytes.internal.BytesInternal;
+import net.openhft.chronicle.jlbh.JLBH;
+import net.openhft.chronicle.jlbh.JLBHOptions;
+import net.openhft.chronicle.jlbh.JLBHTask;
+
+public class ContentEqualsJLBHTest {
```

Comment:
cool. We should add to `run-benchmarks` profile so we can see in TC graphs. You will also need to add
```
    @Override
    public void complete() {
        TeamCityHelper.teamCityStatsLastRun(this.getClass().getSimpleName(), jlbh, iterations, System.out);
    }
```

Reply:
It would be good if this test could execute the old way and also the new way so we can see the difference

Reply:
I'll send you the email with the results.